Update azure.md (#26479)
DataDog/documentation: content/en/cloud_cost_management/azure.md

{{% site-region region="us3" %}}
**Notes**:

> - If you are using Datadog's **US3** site, you may have set up the Datadog Azure Native integration using the recommended [Datadog Resource method][1] through the Azure Portal. To support Cloud Cost Management, you need to [create an App Registration][2], which is in addition to setting up the Datadog Azure Native integration. Both will be needed, the Azure Native integration method will provide the main integration benefits and the App Registration will only be needed for CCM to be setup.

Reviewer: Is this optional? "May" implies that it's not required.

```suggestion
- If you are using Datadog's **US3** site, you may have set up the Datadog Azure Native integration using the recommended [Datadog Resource method][1] through the Azure Portal. To enable Cloud Cost Management, you need to [create an App Registration][2] in addition to setting up the Datadog Azure Native integration. Both are required: the Azure Native integration method provides the main integration benefits, while the App Registration is necessary for setting up CCM.
```
